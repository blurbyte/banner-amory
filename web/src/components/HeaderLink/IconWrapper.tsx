import styled from 'styled-components';

const IconWrapper = styled.span`
  position: absolute;
  z-index: 9999;
  top: 0.8rem;
  left: -1.8rem;
`;

export default IconWrapper;
