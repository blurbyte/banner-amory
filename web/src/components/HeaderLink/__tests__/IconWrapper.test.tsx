import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import IconWrapper from '../IconWrapper';

test('renders correctly', () => {
  const component = renderer.create(<IconWrapper />);
  expect(component.toJSON()).toMatchSnapshot();
});
