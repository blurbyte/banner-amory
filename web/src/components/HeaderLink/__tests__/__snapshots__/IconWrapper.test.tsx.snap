// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  position: absolute;
  z-index: 9999;
  top: 0.8rem;
  left: -1.8rem;
}

<span
  className="c0"
/>
`;
